website/lib/styles.tsx

```diff
@@ -36,6 +36,11 @@ const styles = {
     'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/css/bootstrap.min.css',
   ]),
 
+  bootstrap5: style([
+    'https://cdnjs.cloudflare.com/ajax/libs/octicons/3.5.0/octicons.min.css',
+    'https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css',
```

Comment:
https://getbootstrap.com/docs/5.0/getting-started/download/#cdn-via-jsdelivr

Reply:
As I can see, [it's available on cdnjs as well](https://cdnjs.com/libraries/twitter-bootstrap/5.0.0-beta3). Could we use this one instead?